crates/biome_js_formatter/src/ts/auxiliary/property_signature_type_member.rs

```diff
@@ -25,7 +25,7 @@ impl FormatNodeRule<TsPropertySignatureTypeMember> for FormatTsPropertySignature
             f,
             [
                 readonly_token.format(),
-                space(),
+                conditional_space(readonly_token.is_some()),
```

Comment:
This one took a while to track down lol. This was previously _always_ adding a space between a token that might not exist (the `readonly`) and the identifier. Now that space is conditional based on whether that `readonly` modifier is there or not.

Without it, the signature would always be generated as `" a: string"`, rather than `"a: string"`, and the latter feels more correct anyway.